### PR TITLE
Style landing page

### PR DESCRIPTION
**Pull request by jasonramirez**
## Style landing page

screen shot 2015-02-19 at 10 30 50 am

screen shot 2015-02-19 at 10 29 03 am

### CODE REVIEW

File: app/assets/stylesheets/patterns/_section.scss [hunk 1]
```diff
@@ -9,6 +9,11 @@
   justify-content: center;
   min-height: em(300);
   text-align: center;
+
+  h1,
```
Comment (Euraldius) [inline app/assets/stylesheets/patterns/_section.scss]: Is it convention to keep the target names on separate lines?

**Comment by Euraldius** [comment]
screen shot 2015-02-18 at 15 09 45

Do we want the Sign out link to appear as it is here? It also looks like the hero image has a small amount of white space on the right side of the browser

**Comment by jasonramirez** [comment]
@Euraldius good call! I didn't have that branch with account creation in here, so that didn't show up on mine. Ill address it!

File: app/assets/stylesheets/patterns/_section.scss (hunk 1 above)
Comment (jasonramirez) [inline app/assets/stylesheets/patterns/_section.scss]: Per hound and our styleguide, each selector gets its own line.

**Comment by Euraldius** [comment]
Failing build :(

**Comment by Euraldius** [comment]
screen shot 2015-02-18 at 17 38 19

The buttons don't interact super well with the notifications.

Also, before we merge this, I'll take a look at why we're getting the irritating secondary notification display

**Comment by jasonramirez** [comment]
@Euraldius moved the nav into the section so it reacts to the relative position of the flash. Nice :eyes: 

screen shot 2015-02-18 at 5 44 24 pm

**Comment by Euraldius** [comment]
Pending green build, this LGTM. :ship: it

File: app/assets/stylesheets/patterns/_navigation.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+.navigation--main {
+  @include position(absolute, 0 0 null 0);
+  text-align: right;
+
+  .button {
+    margin: 1em;
+  }
+
+  .button + .button {
```
Comment (Euraldius) [inline app/assets/stylesheets/patterns/_navigation.scss]: @jasonramirez Not sure what this rule means so I didn't fix it...

File: app/assets/stylesheets/base/_variables.scss [hunk 3]
```diff
@@ -1,8 +1,9 @@
 // Typography
 $sans-serif: $helvetica;
-$serif: $georgia;
-$base-font-family: $sans-serif;
-$header-font-family: $base-font-family;
+$serif: "Lora";
+$slab: "Roboto Slab";
```
Comment (Euraldius) [inline app/assets/stylesheets/base/_variables.scss]: This is a great name

**Comment by Euraldius** [comment]
Hound has a comment, but otherwise LGTM

File: app/assets/stylesheets/patterns/_navigation.scss (hunk 2 above)
Comment (jasonramirez) [inline app/assets/stylesheets/patterns/_navigation.scss]: If I merge them, then the effect is different then what we intended. A good merge would be:

```
.button {
  margin: 1em;

  & + .button {
    margin-left: 2em; 
  }
}
```

But that is more nesting then we need. I think this is a pretty poor comment, but I could be talked out of it.

**Pull request merged**